pages/main-page.ts

```diff
@@ -0,0 +1,22 @@
+export class MainPage {
+  readonly page: Page;
```

Comment:
you are missing to put as readonly the rest of properties of the POM, setStorage, etc.

Reply:
updated.